components/Skills.tsx

```diff
@@ -0,0 +1,53 @@
+import Image from "next/image";
+import SectionTitle from "./SectionTitle";
+import LightContainer from "./containers/Light";
+import { skills } from "@/constants";
+import {
+  TooltipProvider,
+  Tooltip,
+  TooltipTrigger,
+  TooltipContent,
+} from "./ui/tooltip";
+
+const Skills = () => {
+  return (
+    <LightContainer>
+      <section className="flex flex-col py-12 md:py-[72px] items-center justify-center">
+        <SectionTitle title="My skills" />
+        <div className="relative -top-14 bg-gradient-to-r from-cyan-400 to-yellow-300 dark:from-cyan-800 dark:via-cyan-400 dark:to-yellow-300 h-4 w-[150px] lg:w-[200px] lg:-top-[56px]" />
+        <div className="flex flex-wrap gap-x-5 md:gap-x-10 gap-y-5 items-center justify-center">
+          {skills.map((skill) => (
+            <div
+              key={skill.label}
+              className="relative w-16 h-16 md:h-20 md:w-20 lg:h-24 lg:w-24"
+            >
+              <Image
+                src="/circle.svg"
+                alt="circle"
+                fill
+              />
```

Comment:
I think it's better to use SVGR here instead of an Image tag as you'll need to change the circle's color in dark mode like shown in the figma design.

Reply:
Is there any difference than doing the following:

src={
     localStorage.theme === "dark"
     ? "/web-icons/mobile-menu-dark.svg"
     : "/web-icons/mobile-menu.svg"
 }

Reply:
In your example you're using two svgs instead of one where you just change the color as needed.